lib/bepaid.ex

```diff
@@ -1,17 +1,20 @@
 defmodule Bepaid do
+  use Application
+
   defmodule Error do
     @moduledoc """
     Defines an exception for Bepaid errors.
     """
     defexception [:message]
   end
 
-  def start_link do
+
+  def start(_type, _args) do
     import Supervisor.Spec, warn: false
 
     children = []
 
     opts = [strategy: :one_for_one, name: Bepaid.Supervisor]
-    {:ok, _pid} =  Supervisor.start_link(children, opts)
```

Comment:
@MikeAndrianov why we do not check if the app has started successfully?

Reply:
Don't remember exactly :| 
I will put it back